frontend/src/app/main/ui/notifications.cljs

```diff
@@ -26,16 +27,16 @@
         inline?      (or (= :inline (:type notification))
                          (= :floating (:position notification)))
         toast?       (or (= :toast (:type notification))
-                         (some? (:timeout notification)))]
+                         (some? (:timeout notification)))
+        content     (or (:content notification) "")]
 
     (when notification
       (cond
         toast?
-        [:& toast-notification
+        [:& toast*
```

Comment:
Components with `*` in the name should always use `[:>` handler instead of `[:&`